src/kernel/include/shared/cos_types.h

```diff
@@ -364,6 +364,26 @@ struct cos_stack_freelists {
 	struct stack_fl freelists[COMP_INFO_STACK_FREELISTS];
 };
 
+/*
+ * enum for defining the beginning set number of keys that all components will
+ * share in common.
+ */
+enum
+{
+	COMP_KEY = 0,
+	GREETING_KEY,
+	/* After this, the key values are free to use as components need them */
+};
+
+struct kv {
```

Comment:
Namespacing.  `cos_config_info_kv`, perhaps?

Namespacing on the enum above as well...